src/test/kotlin/uk/gov/justice/digital/hmpps/prisonregister/integration/GetPrisonNames.kt

```diff
@@ -78,6 +80,63 @@ class GetPrisonNames : IntegrationTest() {
     }
   }
 
+  /**
```

Comment:
CsvSource to parametize the tests with the given results as you have suggested @jnioche-jd